onyo/shell_completion/zsh/_onyo

```diff
@@ -121,6 +121,7 @@ _onyo() {
                 args+=(
                     '(- : *)'{-h,--help}'[show this help message and exit]'
                     '(-t --template)'{-t,--template}'[template to seed the new assets]:TEMPLATE:_path_files -W "$(_template_dir)" -g "*(.)"'
+                    '(-c --clone)'{-c, --clone}'[asset path to clone from]:CLONE:_files -W "$(_onyo_dir)"'
```

Comment:
You have a spurious space here: `{-c, --clone}` -> `{-c,--clone}`

We're mixing compsys syntax with brace expansion here. So it can be a bit odd.

Reply:
Thx much!